src/main/scala/com/campudus/tableaux/database/model/TableauxModel.scala

```diff
@@ -114,17 +128,29 @@ class TableauxModel(override protected[this] val connection: DatabaseConnection)
     _ <- cellStruc.update(tableId, columnId, rowId, value)
   } yield Cell(column.asInstanceOf[B], rowId, value)
 
-  def insertLinkValue(tableId: IdType, columnId: IdType, rowId: IdType, value: (IdType, IdType)): Future[Cell[Link[Any], LinkColumnType[Any]]] = for {
+  def handleLinkValues(tableId: IdType, columnId: IdType, rowId: IdType, from: IdType, tos: Either[IdType, Seq[IdType]]): Future[Cell[Link[Any], LinkColumnType[Any]]] =
+    tos match {
+      case Left(to) => addLinkValue(tableId, columnId, rowId, from, to)
+      case Right(toList) => insertLinkValues(tableId, columnId, rowId, from, toList)
+    }
+
+  def insertLinkValues(tableId: IdType, columnId: IdType, rowId: IdType, from: IdType, tos: Seq[IdType]): Future[Cell[Link[Any], LinkColumnType[Any]]] = for {
     linkColumn <- getColumn(tableId, columnId).asInstanceOf[Future[LinkColumnType[Any]]]
-    _ <- cellStruc.updateLink(linkColumn.table.id, linkColumn.id, value)
+    _ <- cellStruc.putLinks(linkColumn.table.id, linkColumn.id, from, tos)
     v <- cellStruc.getLinkValues(linkColumn.table.id, linkColumn.id, rowId, linkColumn.to.table.id, linkColumn.to.id)
-  } yield Cell(linkColumn, rowId, Link(List((value._2, v))))
+  } yield Cell(linkColumn, rowId, Link(linkColumn.to.id, v))
+
+  def addLinkValue(tableId: IdType, columnId: IdType, rowId: IdType, from: IdType, to: IdType): Future[Cell[Link[Any], LinkColumnType[Any]]] = for {
+    linkColumn <- getColumn(tableId, columnId).asInstanceOf[Future[LinkColumnType[Any]]]
+    _ <- cellStruc.updateLink(linkColumn.table.id, linkColumn.id, from, to)
+    v <- cellStruc.getLinkValues(linkColumn.table.id, linkColumn.id, rowId, linkColumn.to.table.id, linkColumn.to.id)
+  } yield Cell(linkColumn, rowId, Link(to, v))
 
   def getAllTables(): Future[TableSeq] = for {
     seqOfTableInformation <- tableStruc.getAll
   } yield {
-    TableSeq(seqOfTableInformation map { case (id, name) => Table(id, name) })
-  }
+      TableSeq(seqOfTableInformation map { case (id, name) => Table(id, name) })
+    }
```

Comment:
Is it possible to get the formatter not intending the `yield` part like this?

It should be:
```
for {
  _ <- something()
} yield {
  somethingElse()
}
```

and not

```
for {
  _ <- something()
} yield {
    somethingElse()
  }
```

This happens to me all the time and I can't find the option in IntelliJ to change this behaviour...